Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM node:16.17.0-alpine3.15 AS generate
+LABEL maintainer="ezTeam <ezteam@couperin.org>"
+
+WORKDIR /homepage
+
+COPY package*.json ./
+RUN npm ci 
+COPY . .
```

Comment:
`COPY . .` while already having a `node_modules` will erase the result of `npm ci` (except if we have `node_modules` in a `.dockerignore`, in this case the first `COPY` is useless)